luatest/assertions.lua

```diff
@@ -387,6 +390,8 @@ function M.assert_str_contains(value, expected, is_pattern, message)
     M.assert_type(value, 'string', nil, 2)
     M.assert_type(expected, 'string', nil, 2)
 
+    log.info('Assert string %s contains %s', value, expected)
```

Comment:
I propose to use the following format:

`` Assert string `%s` contains `%s` ``

Also, looks like it should be at the very beginning of the function.